src/Mailtrap.php

```diff
@@ -72,23 +70,32 @@ public function _initialize()
      *
      * @param \Codeception\TestCase $test
      */
+
+    /*
     public function _after(\Codeception\TestCase $test)
     {
-        if ($this->config['cleanup']) {
-            $this->cleanInbox();
-        }
+        $this->cleanInbox();
     }
+    */
 
     /**
      * Clean all the messages from inbox.
      *
      * @return void
      */
+
     public function cleanInbox()
     {
         $this->client->patch("inboxes/{$this->config['inbox_id']}/clean");
     }
 
+    public function getTestEmailAddress()
+    {
+        $testEmailAddress = $this->config['testEmailAddress'];
+
+        return $testEmailAddress;
+    }
```

Comment:
Providing a default in the config would be preferred. Also, the use of a temporary variable is not necessary here and could be avoided: 

``` 
public function getTestEmailAddress()
{
    return $this->config['testEmailAddress'];
}
```